samples/apps/logging/js/src/logging.js

```diff
@@ -15,6 +15,20 @@ function get_id_from_query(parsedQuery) {
   return ccf.strToBuf(parsedQuery.id);
 }
 
+function get_scope(parsedQuery) {
+  return parsedQuery.scope === undefined ? undefined : parsedQuery.scope;
+}
+
+function public_records(store, scope = undefined) {
```

Comment:
Isn't undefined the default default?

Reply:
Possible - lack of JS knowledge, I'll look it up.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters